packages/suitesparse/meta.yaml

```diff
@@ -0,0 +1,43 @@
+package:
+  name: suitesparse
+  version: 1.0.0
+
+source:
+  sha256: 1480acb174081599604f5e894bbe8a1a551de5904eef6ea67f33b25f884cd74f
+  url: https://gabrielfougeron.github.io/files/suitesparse.zip
```

Comment:
Please change this to point to 
1. preferably the upstream release: https://github.com/DrTimothyAldenDavis/SuiteSparse/archive/refs/tags/v5.11.0.zip
2. failing that, a fork of SuiteSparse: https://github.com/gabrielfougeron/SuiteSparse
it's a bit difficult to work with it if it isn't in a github repo. Eventually we want to have a release version + some patch files, but while working on the PR it is fine to point at a personal fork.